q2_feature_table/plugin_setup.py

```diff
@@ -94,6 +94,37 @@
 )
 
 plugin.methods.register_function(
+    function=q2_feature_table.group,
+    inputs={'table': FeatureTable[Frequency]},
+    parameters={
+        'mode': Str % Choices({'sum', 'median-ceiling', 'mean-ceiling'}),
+        'metadata': MetadataCategory,
+        'axis': Str % Choices({'sample', 'feature'})
+    },
+    outputs=[
+        ('grouped_table', FeatureTable[Frequency])
+    ],
+    input_descriptions={
+        'table': 'The table to group samples or features on.'
+    },
+    parameter_descriptions={
+        'mode': 'How to combine observation frequencies within group.',
```

Comment:
This is probably confusing terminology, is there a better word for the cells of a contingency table? Also, I should describe the meaning of the modes, now that they aren't as clear-cut

Reply:
Googling around I found "joint frequency" which kind of makes sense.